napari/_qt/qt_viewer.py

```diff
@@ -478,6 +484,8 @@ def _save_layers_dialog(self, selected=False):
                         error_messages=error_messages,
                     )
                 )
+            else:
+                update_save_history(saved[0])
```

Comment:
I recently moved this save history function to here to make sure it updated the history when a file was successfully saved.